docs/index.rst

```diff
@@ -66,7 +69,7 @@ Output the data in your views.
     def users():
         all_users = User.all()
         result = users_schema.dump(all_users)
-        return jsonify(result.data)
+        return jsonify({"users": result.data})
```

Comment:
For now, I'm going to just remove the envelope so that the return value is equivalent to `users_schema.jsonify(all_users)`.